pkgs/by-name/ar/art/package.nix

```diff
@@ -0,0 +1,84 @@
+{ lib
+, stdenv
+, fetchFromBitbucket
+, cmake
+, pkg-config
+, wrapGAppsHook3
+, makeWrapper
+, pixman
+, libpthreadstubs
+, gtkmm3
+, libXau
+, libXdmcp
+, lcms2
+, libiptcdata
+, fftw
+, expat
+, pcre
+, libsigcxx
+, lensfun
+, librsvg
+, libcanberra-gtk3
+, exiv2
+, exiftool
+, mimalloc
+}:
+
+stdenv.mkDerivation rec {
+  pname = "art";
+  version = "1.22.1";
+
+  src = fetchFromBitbucket {
+    owner = "agriggio";
+    repo = "art";
+    rev = version;
+    hash = "sha256-f6SnTvMelJaPGNeGboI34RvWXcJatEi1G6vfAdDFy8A=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    wrapGAppsHook3
+  ];
+
+  buildInputs = [
+    pixman
+    libpthreadstubs
+    gtkmm3
+    libXau
+    libXdmcp
+    lcms2
+    libiptcdata
+    fftw
+    expat
+    pcre
+    libsigcxx
+    lensfun
+    librsvg
+    exiv2
+    exiftool
+    libcanberra-gtk3
+    mimalloc
+  ];
+
+  cmakeFlags = [
+    "-DPROC_TARGET_NUMBER=2"
+    "-DCACHE_NAME_SUFFIX=\"\""
+  ];
+
+  CMAKE_CXX_FLAGS = toString [
+    "-std=c++11"
+    "-Wno-deprecated-declarations"
+    "-Wno-unused-result"
+  ];
+  env.CXXFLAGS = "-include cstdint"; # needed at least with gcc13 on aarch64-linux
+
+  meta = with lib; {
```

Comment:
Better to avoid `with lib`

Reply:
Can you tell me why?

Reply:
https://github.com/NixOS/nixpkgs/issues/208242

Reply:
Updated!